snippets/nodejs/netbackup-module/netbackup.js

```diff
@@ -0,0 +1,138 @@
+// =============================================================
+// netbackup.js
+// +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+// This file contains modules to call NetBackup APIs.
+// =============================================================
+
+var https = require('https');
+var fs = require('fs');
+
+var exportedMethod = {};
+
+// To disable certificate/ssl verification, uncomment following line
+// process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+// TODO save ca certificate at run time and refer that location
+var CA_CERT_PATH
```

Comment:
Does this mean script can only be run on NetBackup master server?

Reply:
I am fixing this by saving ca certificate on local trust store.